docker/docker-compose.yml

```diff
@@ -13,6 +13,7 @@ services:
       endpoint_mode: dnsrr
     volumes:
       - db-data:/var/lib/postgresql/data/
+      - ./enable_btree_gist.sh:/docker-entrypoint-initdb.d/enable_btree_gist.sh
```

Comment:
This looks better to me 👍  Have you tested to make sure it loads the script correctly?

Reply:
Yes, but CI is marked as fail due to #2097